apps/website/src/app/page.tsx

```diff
@@ -617,7 +617,7 @@ function HeroSection() {
     const copyIcon =
         <div className={"w-fit h-fit z-10 cursor-pointer"} onClick={() => {
             setCopiedToClipboard(true);
-            navigator.clipboard.writeText("npm init @nameguard/nameguard@latest");
+            navigator.clipboard.writeText("npm init @namehash/nameguard@latest");
```

Comment:
install?

Reply:
idk, @notrab?

Reply:
I have a PR open already that addresses this